extensions/big-data-cluster/src/bigDataCluster/localizedConstants.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I remembered I did this once and @kevcunnane left a comment saying we should avoid this. @kevcunnane do you recall the reason?